pkg/tnf/identifier/identifiers.go

```diff
@@ -51,6 +51,7 @@ const (
 	readRemoteFileIdentifierURL           = "http://test-network-function.com/tests/readRemoteFile"
 	uncordonNodeIdentifierURL             = "http://test-network-function.com/tests/node/uncordon"
 	checkSubscriptionIdentifierURL        = "http://test-network-function.com/tests/operator/check-subscription"
+	podantiaffinityIdentifierURL          = "http://test-network-function.com/tests/podantiaffinity"
```

Comment:
As I mentioned in standup, it's better to keep the test case name more high level and customer oriented, e.g. testPodHighAvailability. The handlers can be named with specific checks, like testPodAntiaffinity, testReplicaCount

Reply:
you mean URL "podantiaffinity" = >  testPodHighAvailability?

Reply:
yes

Reply:
test name/url not updated